db/migrate/20170201124248_add_defaults_to_timestamps_in_measurements.rb

```diff
@@ -0,0 +1,35 @@
+class AddDefaultsToTimestampsInMeasurements < ActiveRecord::Migration[5.0]
+  def up
+    change_table :measurements do |t|
+      %i(created_at updated_at).each do |col|
+        t.change col, :datetime, null: true, default: -> { 'now()' }
+      end
+    end
+
+    execute <<-__END__
```

Comment:
Can probably indent this when using `<<-`

I also like to use `SQL` as the delimiter. That way IntelliJ (I think some others too) can syntax highlight within the heredoc.